src/components/organisms/bucket-add-field/BucketAddField.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { FC, memo, useState } from "react";
+import { createShema } from "./BucketAddFieldSchema";
+import Button from "components/atoms/button/Button";
+import useInputRepresenter from "custom-hooks/useInputRepresenter";
+import Tab from "components/atoms/tab/Tab";
+import FlexElement from "components/atoms/flex-element/FlexElement";
+import FluidContainer from "components/atoms/fluid-container/FluidContainer";
+import Icon from "components/atoms/icon/Icon";
+import styles from "./BucketAddField.module.scss";
+import { TypeInputType } from "../../../custom-hooks/useInputRepresenter";
+import Modal from "components/atoms/modal/Modal";
+import { TypeModal } from "utils/interface";
+
+type TypeBucketAddField = {
+  name: string;
+  type: TypeInputType;
+  isOpen?: boolean;
+  modalProps?: TypeModal;
+};
+
+const BucketAddField: FC<TypeBucketAddField> = ({ name, type, isOpen, modalProps }) => {
+  const isInnerFieldsType = type === "object" || type === "array";
```

Comment:
this approach is cleaner
```suggestion
  const isInnerFieldsType = ["object", "array"].includes(type);
```

Reply:
Done